dashboard/app/models/script.rb

```diff
@@ -517,10 +518,6 @@ def cs_in_a?
     name.match(Regexp.union('algebra', 'Algebra'))
   end
 
-  def has_lesson_plan?
-    k5_course? || k5_draft_course? || %w(msm algebra algebraa algebrab cspunit1 cspunit2 cspunit3 cspunit4 cspunit5 cspunit6 csp1 csp2 csp3 csp4 csp5 csp6 csppostap cspoptional csp3-research-mxghyt csd1 csd2 csd3 csd4 csd5 csd6 csp-ap text-compression netsim pixelation frequency_analysis vigenere applab-intro csp-explore csp-create).include?(name)
```

Comment:
Every script referenced in this conditional was updated, except for `msm.script` which I can't find - I don't think it exists anymore.